lib/broccoli/ember-app.js

```diff
@@ -38,6 +38,8 @@ var merge         = require('lodash/object/merge');
 var omit          = require('lodash/object/omit');
 var Funnel        = require('broccoli-funnel');
 
+var deprecate     = require('../utilities/deprecate');
```

Comment:
`deprecateUI` itself is deprecated. :stuck_out_tongue_winking_eye: This should probably go through `ui.writeDeprecateLine`.

https://github.com/ember-cli/ember-cli/blob/master/lib/ui/index.js#L129